src/www/wizard.php

```diff
@@ -885,7 +885,7 @@ function showchange() {
 				echo $field['typehint'];
 			}
 			if($field['warning'] <> "") {
-				echo "<br /><b><font color=\"red\">" . $field['warning'] . "</font></b>";
+				echo "<br /><b><span style='color: red;'>" . $field['warning'] . "</span></b>";
```

Comment:
Use a suitable bootstrap class instead

Reply:
The most suitable would be `text-warning`, though that would be orange, not red. Is that OK?

Reply:
for this case text-alert would be more appropriate despite the "warning" designation

Reply:
text-alert is not defined in any CSS file. Did you mean "text-info"?

Reply:
sorry, I meant text-danger

Reply:
Hm, would that really be more appropriate? I thought this field showed warnings that weren't that urgent? To me, it seems that classifying them as 'danger' would be a bit overkill.

Reply:
ok, let's back up a little :) how often is 'warning' used in the wizard xml files?

Reply:
I cannot find any occurrence in `wizard/system.xml` and `wizard/openvpn.xml`.

If you really think text-danger is more appropriate, I'll just go with that. :)

Reply:
if nothing uses it the point is moot. better remove the "functionality" ?

Reply:
You know the system better than I do. Are the two XML files I mentioned the only places this "warning" could come from?

I'll leave it to you to decide whether to remove it. :)

Reply:
yeah, only those two are left with the wizard system deprecated and in need of a better replacement eventually. please remove then, thanks!